src/aurora/ltrfile.cpp

```diff
@@ -0,0 +1,117 @@
+//
+// Created by patrick on 02.05.18.
+//
+
+#include "src/common/readstream.h"
+#include "src/common/strutil.h"
+
+#include "src/aurora/ltrfile.h"
+#include "src/aurora/resman.h"
+
+static const uint32 kLTRID  = MKTAG('L', 'T', 'R', ' ');
+static const uint32 kVersion10 = MKTAG('V', '1', '.', '0');
```

Comment:
Align those properly, please, or take out the second space between `kLTRID` and `=`